ebpfcore/ebpf_drv.c

```diff
@@ -380,3 +387,32 @@ ebpf_driver_get_device_object()
 {
     return _ebpf_driver_device_object;
 }
+
+NTSTATUS
+_ebpf_driver_query_volume_information(_In_ WDFDEVICE device, _Inout_ PIRP irp)
+{
+    NTSTATUS status;
+    PIO_STACK_LOCATION irp_stack_location;
```

Comment:
recommend using `IRP*` instead of `PIRP` and `IO_STACK_LOCATION*` instead of `PIO_STACK_LOCATION`